### PR TITLE
SITES-225: Block acsf_openid module

### PR DESCRIPTION
**Pull request by jbickar**
# READY FOR REVIEW

# Summary
- Disable `acsf_openid` during migration, so that sites that use local Drupal accounts can use them.

# Needed By (Date)
- Next week (week of July 16th) should be fine

# Criticality
- How critical is this PR on a 1-10 scale? 2/10
- Affects only one site as far as we know

# Steps to Test

1. Check out `master`
2. Clone `supri-b` to `dev` or `test`
3. Go to `affiliates/affiliate-program`
4. See no login link in the first sidebar
5. Check out this branch
6. Go to `affiliates/affiliate-program`
7. See the login link in the first sidebar
8. Optionally, create a local Drupal user account with the role "affiliate" and ensure that you can log in and see the affiliate links in the first sidebar

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules? ACSF

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/SITES-225

### CODE REVIEW

File: group_vars/all.yml [hunk 1]
```diff
@@ -28,3 +28,8 @@ stack_id: "2"
 sitefactory_environment: ""
 stanford_environment: ""
 drush_environment: ""
+
+# This variable (ignore_updb_errors should be set to an empty string by
+# defaults, in order to avoid "undefined variable" error messages. It can be
+# set on a per-site basis in the inventory file.
+ignore_updb_errors: ""
```
[the remark this answers is not in the document]
Comment (jbickar) [reply group_vars/all.yml]: Right?

**Comment by jbickar** [comment]
@sherakama hold on this PR until tomorrow. I thought of one more thing: I want to pull the second updb into its own task and only run it if ignore_updb_errors=true

**Comment by jbickar** [comment]
Oh. Stupid me. I thought this was the other PR, #41. Hold on this one until we get socorro merged. #41 should be GTG now.

File: roles/upload-site/tasks/main.yml [hunk 2]
```diff
@@ -84,18 +84,16 @@
...
 
 - name: Do post-database-restore tasks
   shell: "{{ drush_alias }} {{ item }}"
   with_items:
-    - "-y en acsf stanford_ssp paranoia"
-    - "ev 'acsf_openid_allow_local_user_logins();'"
+    - "-y en acsf acsf_helper stanford_ssp paranoia"
     - 'ev "_paranoia_remove_risky_permissions();"'
-    - "-y dis stanford_afs_quota"
+    - "-y dis stanford_afs_quota acsf_openid openid"
```
Comment (sherakama) [inline roles/upload-site/tasks/main.yml]: I had to disable with `$disable_dependents = FALSE` in the installation scripts and I can't tell if drush does that.

https://github.com/SU-SWS/stanford_install_tasks/blob/7.x-1.x/Environment/ACSF/Install/EnableModules.php#L24
https://api.drupal.org/api/drupal/includes%21module.inc/function/module_disable/7.x
https://drushcommands.com/drush-7x/pm/pm-disable/

Also, we will want to uninstall these modules and not just disable.

**Comment by jbickar** [comment]
Good call on uninstalling.

File: roles/upload-site/tasks/main.yml (hunk 2 above)
Comment (jbickar) [reply roles/upload-site/tasks/main.yml]: drush disables dependent modules. So we could just go with disabling `openid` here but I would like to be explicit.

**Comment by jbickar** [comment]
Give 'er a whirl now.

**Pull request merged**